packages/ember-data/lib/adapters/build-url-mixin.js

```diff
@@ -27,7 +27,7 @@ var get = Ember.get;
   @class BuildURLMixin
   @namespace DS
 */
-var BuildURLMixin = Ember.Mixin.create({
+export default Ember.Mixin.create({
```

Comment:
if cases exist where the path is not strongly related, that should be considered a bug. If other cases exist where the path isn't searchable, that also appears like a bug.

Even without source maps, searching the built-output for the on-disk location will still reveal the AMD module, as we opt to fully named and resolved modules.

In addition, searching this way can also reveal importers.